config/interceptor/deployment.yaml

```diff
@@ -70,6 +70,7 @@ spec:
           allowPrivilegeEscalation: false
           readOnlyRootFilesystem: true
           runAsNonRoot: true
+          runAsUser: 65532
```

Comment:
I'm not totally sure if we should set this value here. I remember problems related with userIds in OpenShift. If I'm not wrong, OpenShift generate random used id ranges for each workload. Being honest, this is not a problem IMO because users can use helm chart to deploy the add-on, but just in case...
Any thought @tomkerkhove ?

Reply:
I set this because the pod was refusing to go up on AKS. I can get the specific error for you latter, but the problem is that on the Dockerfile we are setting the user to this value and on the K8s manifest we are setting `runAsNonRoot: true`, this combination mandates that we also specify `runAsUser` here.

Reply:
For future reference, the issue regarding runAsUser is kedacore/keda#3012 and it was fixed by kedacore/keda#3015.

In core we have two different security contexts, one for the container and another one for the pod. Replicating the same structure here, the problem I was experiencing is gone and we don't need to set runAsUser anymore.